azext_iot/product/_help.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Product Certification commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_help():
+    helps["iot product"] = """
+        type: group
+        short-summary: Manage device testing for product certification
+    """
+# certification requirements
+    helps["iot product requirement"] = """
+        type: group
+        short-summary: Manage product certification requirements
+    """
+    helps["iot product requirement list"] = """
+        type: command
+        short-summary: Get certification requirements
```

Comment:
Are users supposed to know all the concepts that will be introduced in the `az iot product` command space? Like me, not being familiar with this service, am not sure what certification requirements are in this context. It will always  benefit users to include more details, even briefly. `long-summary` is good for longer explanations (if needed).

Reply:
The best CLI experiences are as self-describing as possible

Reply:
Added a long summary to this command. Currently the service returns supported attestation methods by badge type. If this is expanded in the future we'll want to update the long-summary